src/api-explorer/v4-0/Request.swagger2.json

```diff
@@ -0,0 +1,3922 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Manage documents used for pre-spend authorizations within Concur Request.",
+    "version": "4.0",
+    "title": "Request"
+  },
+  "host": "www.concursolutions.com",
```

Comment:
Hello, it seems that other documentation use "www.concursolutions.com" as host. However, the page https://developer.concur.com/api-explorer/ declares " https://us.api.concursolutions.com" for OAuth2 framework. Which host does the api use?

Reply:
Check https://developer.concur.com/api-explorer/ API v3 is using the pre2017 authentication therefore the path was https://www.concursolutions.com/api/{version}
Now we have Oauth2 authentication, therefore the path should be https://us.api.concursolutions.com

You should clarify with API team what is the approach to different DCs : 
1) we provide swagger only for us DC ?
2) we can or should have one swagger per DC ?
3) API gateway is able to do the magic of forwarding https://us.api.concursolutions.com to the right DC ?